rj_param_utils/include/rj_param_utils/global_param.hpp

```diff
@@ -0,0 +1,61 @@
+/** @file */
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp/node.hpp>
+#include <rj_param_utils/param.hpp>
+#include <rj_param_utils/ros2_param_provider.hpp>
+
+/**
+ * @brief Create node to store global parameters.
+ */
+auto global_param_node = rclcpp::Node::make_shared("global_parameter_node");
```

Comment:
This should be in a `main()` function for the global parameter node. Put it in `global_param_server.cpp` or something (for now, each ROS node is a separate binary).